dydx/dydxViews/dydxViews/_v4/Primers/dydxNotificationPrimerView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  dydxNotificationPrimerView.swift
+//  dydxUI
+//
+//  Created by Rui Huang on 11/09/2024.
+//  Copyright © 2024 dYdX Trading Inc. All rights reserved.
+//
+
+import SwiftUI
+import PlatformUI
+import Utilities
+
+public class dydxNotificationPrimerViewModel: PlatformViewModel {
+    @Published public var ctaAction: (() -> Void)?
```

Comment:
general question, why do we show a primer if "ok" is the only action? Should we add like a "not now" option? Otherwise, the primer is just an extra step for the user to say "no"

Reply:
User can drag down the half sheet to dismiss, so it's the same as "not now".   

The primer serves as a value prop to the subsequent system notification toggle which is quite generic.  The idea is that with the value prop the user will be more likely to allow notification in the next step.